### DISCUSSION TITLE
Login with dreame cloud instead of mihome cloud

### DISCUSSION THREAD

**Discussion by ghassanjankeel** [Tasshack/dreame-vacuum#85] [Q&A] [answered]
I have a dreametech l10s ultra, I use it with dreamehome app mostly for the added features like live view cam and more frequent SW updates.
is there a way for it to be added to Home assistant without having it in Xiaomi mihome app?

**Comment by Tasshack** [comment, marked as answer]
Please check the related issue about this https://github.com/Tasshack/dreame-vacuum/issues/39

**Reply by ghassanjankeel** [reply]
Thanks for the great work, will wait until the camera is supported in your future release, then will switch to Xiaomi cloud

**Comment by Tasshack** [comment]
https://github.com/Tasshack/dreame-vacuum/discussions/109
